recipes-devtools/python/python3-jepture_git.bb

```diff
@@ -0,0 +1,34 @@
+SUMMARY = "A simple python library to quickly capture images on the Jetson."
+LICENSE = "Proprietary & BSD-3-Clause & MIT"
+LIC_FILES_CHKSUM = " \
+    file://LICENSE.md;md5=4f0a72805ccc91536540a1cf89a935ec \
+    file://jetson_multimedia_api/usr/src/jetson_multimedia_api/argus/LICENSE.TXT;md5=271791ce6ff6f928d44a848145021687 \
+"
+
+COMPATIBLE_MACHINE = "(tegra)"
+
+L4T_DEB_SOCNAME = "t186"
+L4T_BSP_DEB_VERSION = "${L4T_BSP_DEB_DEFAULT_VERSION_T186}"
+PKG_VER = "${L4T_VERSION}${@l4t_bsp_debian_version_suffix(d)}"
+SRC_SOC_DEBS = "nvidia-l4t-jetson-multimedia-api_${PKG_VER}_arm64.deb;subdir=git/jetson_multimedia_api"
```

Comment:
@brgl 
Why we need `nvidia-l4t-jetson-multimedia-api` here ? 
FYI, we have `tegra-mmapi` recipes in `meta-tegra` ?
I think you have to add `require tegra-mmapi-32.7.2.inc` and remove all stuff related to `nvidia-l4t-jetson-multimedia-api_XXXXXX_arm64.deb`

Reply:
I think it's OK... the include file has some other settings that wouldn't be appropriate here.